vmdb/product/toolbars/ems_infra_center_tb.yaml

```diff
@@ -22,6 +22,11 @@
       :text: 'Edit this #{ui_lookup(:table=>"ems_infra")}'
       :title: 'Edit this #{ui_lookup(:table=>"ems_infra")}'
       :url: '/edit'
+    - :button: ems_infra_scale
+      :image: edit
+      :text: 'Scale this #{ui_lookup(:table=>"ems_infra")}'
+      :title: 'Scale this #{ui_lookup(:table=>"ems_infra")}'
```

Comment:
@Ladas, @rwsu : I have just merged a change that we use ERB for the toolbars. Sorry for the confusion. But we got rid of some `eval` calls whis way.